pkg/cli/testutils/kubeconfig.go

```diff
@@ -0,0 +1,139 @@
+// Package testutils provides test utility functions for the CLI package
+package testutils
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// CreateTestKubeconfig creates a temporary kubeconfig file for testing
+func CreateTestKubeconfig(t *testing.T) (string, func()) {
+	t.Helper()
+
+	// Create a temporary kubeconfig file
+	tmpFile, err := os.CreateTemp("", "kubeconfig-")
+	if err != nil {
+		t.Fatalf("Failed to create temp file: %v", err)
+	}
+
+	// Write test kubeconfig content
+	kubeconfigContent := `apiVersion: v1
+clusters:
+- cluster:
+    server: http://test-host
+  name: test-cluster
+contexts:
+- context:
+    cluster: test-cluster
+    user: test-user
+  name: test-context
+current-context: test-context
+kind: Config
+preferences: {}
+users:
+- name: test-user
+`
+	if _, err := tmpFile.WriteString(kubeconfigContent); err != nil {
+		t.Fatalf("Failed to write kubeconfig: %v", err)
+	}
+	if err := tmpFile.Close(); err != nil {
+		t.Fatalf("Failed to close temp file: %v", err)
+	}
+
+	cleanup := func() {
+		_ = os.Remove(tmpFile.Name()) // Ignore error for test cleanup
+	}
+
+	return tmpFile.Name(), cleanup
+}
+
+// CreateTestKubeconfigForExecution creates a temporary kubeconfig file for functional testing
+func CreateTestKubeconfigForExecution(t *testing.T) (string, func()) {
```

Comment:
`CreateTestKubeconfigForExecution` and `CreateTestKubeconfig` appear functionally identical. Can we consolidate the two?